tests/test_flow/test_logging.py

```diff
@@ -112,9 +119,48 @@ def log_unpickleable_value():
 
     assert builder.build().get("log_unpickleable_value") == 5
 
-    log_checker.expect(
+    log_checker.expect_all(
         "Computing  log_unpickleable_value() ...",
         "Cannot pickle me",
         "Logging unpickleable class: Cannot pickle me",
         "Computed   log_unpickleable_value()",
     )
+
+
+@pytest.mark.no_parallel
+@pytest.mark.needs_aip
+def test_log_aip(aip_builder, log_checker):
+    builder = aip_builder
+
+    builder.assign("x", 1)
+
+    @builder
+    @bn.aip_task_config("n1-standard-4")
+    def x_plus_one(x):
+        return x + 1
+
+    @builder
+    @bn.aip_task_config("n1-standard-8")
+    def x_plus_two(x_plus_one):
+        return x_plus_one + 1
+
+    flow = builder.build()
+
+    assert flow.get("x_plus_one") == 2
+
+    log_checker.contains_substrings(
```

Comment:
I assume we use substrings because we can't predict the full AIP url? Could we use regexes here to make it clear what part we can't predict?